src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -48,7 +48,7 @@
 // advance which version they will stop working with
 // Don't go over 32767 for any of these
 #define WALLET_RPC_VERSION_MAJOR 1
-#define WALLET_RPC_VERSION_MINOR 7
+#define WALLET_RPC_VERSION_MINOR 11
```

Comment:
Why did you bump it so high? 
That state only rpc major.minor version and do not take Hard-Fork version at all

Reply:
// When making *any* change here, bump minor
// If the change is incompatible, then bump major and set minor to 0
// This ensures WALLET_RPC_VERSION always increases, that every change
// has its own version, and that clients can just test major to see
// whether they can talk to a given wallet without having to know in
// advance which version they will stop working with
// Don't go over 32767 for any of these

Reply:
4 commits -> 4 bumps.

Reply:
Good enough for me then :)